imaging_tracking/settings.py

```diff
@@ -132,6 +131,6 @@
 
 # Email
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
-EMAIL_HOST = "mail.sanger.ac.uk"
-EMAIL_PORT = 25
+EMAIL_HOST = os.getenv('EMAIL_HOST')
+EMAIL_PORT = os.getenv('EMAIL_PORT')
```

Comment:
Since we are using deafult... do you want to remove this one (`EMAIL_PORT = os.getenv('EMAIL_PORT')`)?

Reply:
If that still works, sure. I'll test now